Crashteroids Starter/Assets/RW/Scripts/Asteroid.cs

```diff
@@ -55,8 +57,31 @@ private void OnCollisionEnter(Collision collision)
     {
         if (collision.gameObject.name == "ShipModel")
         {
-            Game.GameOver();
-            Destroy(gameObject);
+           Game.GameOver();
+           Destroy(gameObject);
         }
     }
+
+    public void LargeAsteroid()
```

Comment:
This function could be named better, from reading it I have no clue what it does